### DISCUSSION TITLE
how to merge cells in table？ is there a demo ？

### DISCUSSION THREAD

**Discussion by netcy** [dolanmiu/docx#872] [Ideas / Requests]
how to merge cells in table？ is there a demo ？

**Comment by netcy** [comment]
![image](https://user-images.githubusercontent.com/5432064/113378513-cd6a8b80-93a9-11eb-9d85-130a810c37c7.png)

**Comment by netcy** [comment]
oh，I find the demo

**Reply by dolanmiu** [reply]
https://github.com/dolanmiu/docx/blob/master/demo/41-merge-table-cells-2.ts
https://github.com/dolanmiu/docx/blob/master/demo/32-merge-and-shade-table-cells.ts
